src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -537,11 +518,9 @@ class GPT2DoubleHeadsModelOutput(ModelOutput):
                 - gpt2-xl: 48
 
     Example::
-
             # Here is an example of a device map on a machine with 4 GPUs using gpt2-xl, which has a total of 48 attention modules:
             model = GPT2LMHeadModel.from_pretrained('gpt2-xl')
             device_map = {0: [0, 1, 2, 3, 4, 5, 6, 7, 8],
```

Comment:
Please don't remove these lines, they're necessary for doc generation and help with readability